test/ts-scenario/config/handlers/sample-transaction-event-handler.ts

```diff
@@ -158,15 +88,8 @@ class SampleTransactionEventHandler {
  * @param {Transaction} transaction The transaction being submitted.
  * @param {Network} network The network where the transaction is being submitted.
  */
-function createTransactionEventHandler(transaction: Transaction, network: Network): SampleTransactionEventHandler {
-	const channel: Client.Channel = network.getChannel();
-	const peers: ChannelPeer[] = channel.getPeersForOrg().filter(hasEventSourceRole);
-	const eventHubs: ChannelEventHub[] = peers.map((peer: ChannelPeer) => channel.getChannelEventHub(peer.getName()));
-	return new SampleTransactionEventHandler(transaction, eventHubs, {});
-}
-
-function hasEventSourceRole(peer: ChannelPeer): boolean {
-	return peer.isInRole('eventSource');
+function createTransactionEventHandler(transaction: Transaction, options: any): SampleTransactionEventHandler {
```

Comment:
This needs to be reverted to a real working usage example. It is exposed as an example in the docs and also confirms that it is possible for end users to implement custom handlers using only the public API.